docs/source/index.rst

```diff
@@ -33,8 +25,70 @@ More infos on Les Fruits Défendus
    harvest-guide/index
 
 
-Indices and tables
-------------------
+What's new in Saskatoon 2022
+----------------------------
+
+The collective worked hard over the last two years to develop an updated version of Saskatoon.
+We fixed many bugs and added a few improvements. 
+
+Changes that affects pick-leaders
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Les Fruits Défendus Guides are now accessible as a public website instead of being PDF documents.
+  Note that the documentation is still a work in progress and the website will be updated during the season. 
+- Saskatoon now sends mails from the following email address: lesfruitsdefendus@riseup.net, incoming emails will be redirected to info@lesfruitsdefendus.org.
+- Unselected pickers receive an email when the harvest passes from status "Date Scheduled" to 'Ready". The template looks like the following::
+   
+   Hi {name}
+
+   We are sorry but enough pickers have already been selected 
+   for the {harvest}. You may still be contacted by the 
+   pick-leader if some of them end up cancelling. 
+   We will do our best to prioritize your participation 
+   next time you submit a request re-using the same email for another harvest.
+
+   Thanks for supporting your community!
+   Yours
+
+   --
+   Saskatoon Harvest System
+   
+  Pick-leader still need to organize with selected pickers.
 
-* :ref:`genindex`
-* :ref:`search`
+
+Changes that mostly affects core members
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- When a pending property has been validated, the tree owner receives a confirmation email. The template looks like the following::
+  
+   Hi {name}
+
+   Your tree subscription has been validated 
+   by a member of Les Fruits Défendus.
+   A pick leader might contact you to plan a 
+   harvest this season.
+
+   Thanks for supporting your community!
+   Yours
+
+   --
+   Saskatoon Harvest System
+
+- The process to register new users has been enhanced such that an "Auth-User" is always created when using this new form: https://saskatoon.lesfruitsdefendus.org/person/create/.
+  You will be asked to select role(s) for this new user. The roles determine the privileges of the user (see https://github.com/LesFruitsDefendus/saskatoon-ng/blob/develop/doc/permissions.org for more infos).
+  Then, use the "Register User" button from the community page to create a password for this user.
+
+- The process to validate new properties has been enhanced such that:
+  
+  - A list of similar properties is listed to help determine if the property already exist in database.
```

Comment:
NB: we should really test this and adjust filtering criteria, especially now that phone numbers are properly formatted using phone fields